### ISSUE TITLE
User Commenting Support

### ISSUE THREAD

**Issue by virtualdisaster** [webgovernor/pykwiki#10]
Static generators such as [pelican](https://github.com/getpelican/pelican) and [nikola](https://github.com/getnikola/nikola) support Disqus, Google+ and Facebook commenting. Will pykwiki offer similar functionality?

**Comment by webgovernor** [comment]
Hi there!

Since the themes are simply Jinja2 HTML files, Disqus, Facebook, Twitter, and any other custom markup can be added. 

Most themes support Facebook and Twitter integration. This is globally defined within the `social.yaml` file, however, theme creators can specify a custom theme options file that may have configuration options for more platforms. For example. the "Blogit" theme supports Disqus comments by setting the Disqus Shortname option in that theme's `options.yaml` file. 

tl;dr - It depends on the theme, but Pykwiki itself theoretically supports all social services, and most built in themes support Facebook and Twitter. See [Included Themes](http://pykwiki.nullism.com/themes.html) for information about themes that I help maintain personally.

Thanks!

**Comment by webgovernor** [comment]
Update: Looking over the archive, I do not see any themes that support Google+ and Facebook **comments**, only Disqus. 

This can be added to your own themes by adding the comment HTML to the post.html theme file.

**Comment by virtualdisaster** [comment]
Ah cool thank you very much for the update and information! You're correct, looks like I assumed they supported other commenting systems, sorry about that. Great work and happy hacking!
